BenchTesting.java

```diff
@@ -0,0 +1,135 @@
+public class BenchTesting {
```

Comment:
Also make sure all of the actual results match exactly with the expected results.

Reply:
Yep 👍🫡

Reply:
Some of the outputs say they expect a simple string with a few characters but instead call a `toString()` method which returns the whole state